forest/shared/src/cli/mod.rs

```diff
@@ -87,6 +87,9 @@ pub struct CliOpts {
     /// Daemonize Forest process
     #[structopt(long)]
     pub detach: bool,
+    /// Enable or disable colored logging in `stdout`
+    #[structopt(long, default_value = "auto", possible_values = &["always", "auto", "never"])]
+    pub color: String,
```

Comment:
It would be more expressive to have the `color` as an enum that implements `FromStr`. Not a huge deal though.

Reply:
I like this, would make it much cleaner